loader/windows/icd_windows.c

```diff
@@ -58,16 +58,11 @@ BOOL adapterAdd(const char* szName, LUID luid)
             newCapacity *= 2;
         }
 
-        WinAdapter* pNewBegin = malloc(newCapacity * sizeof(*pWinAdapterBegin));
+        WinAdapter* pNewBegin = realloc(newCapacity * sizeof(*pWinAdapterBegin));
```

Comment:
Can you check that this is the correct use of `realloc`?  I think this is incorrect and it's causing CI builds to fail.

Reply:
This is fixed and it looks like CI builds are passing now.